src/components/pixi-wind-layer.tsx

```diff
@@ -38,6 +38,11 @@ const arrowTexture = (() => {
   return tex;
 })();
 
+// Use this function to tweak visual length of the wind arrows.
+const arrowLengthFunc = (vec: IVector) => {
+  return Math.pow(4 * Math.sqrt(vec.u * vec.u + vec.v * vec.v), 0.55) + 2;
```

Comment:
tiny suggestion - we have a utils class for other vector pieces, do we use magnitude anywhere else? if so it might be nice to have a helper function.

Reply:
Note that this is not a standard length / magnitude. It includes some weird scaling, which is only used by the rendering code to avoid super-long arrows. So, it's very specific to this particular class.